--- mod_antihak/Makefile.am~	Tue Oct  9 11:44:04 2001
+++ mod_antihak/Makefile.am	Wed Jan  9 00:57:56 2002
@@ -1,7 +1,7 @@
 bin_PROGRAMS = mod_antihak.so
 mod_antihak_so_SOURCES = mod_antihak.c
 
-mod_antihak.so: mod_antihak.c
+mod_antihak.so$(EXEEXT): mod_antihak.c
 	@APACHE_APXS@ -c -I@APACHE_DIR@/include \
 	@MYSQL_INC@ -I@EXTRA_INCLUDE@ \
 	-o mod_antihak.so @MYSQL_LDADD@ @LIBS@ @MYSQL_LIB@ ${mod_antihak_so_SOURCES}
